platforms/android/com/crittercism/plugin/CDVCrittercism.java

```diff
@@ -57,16 +57,22 @@ public void initialize(CordovaInterface cordova, CordovaWebView webView) {
         Context context = this.cordova.getActivity();
         packageName = context.getPackageName();
         Resources resources = context.getResources();
-        String crAppID = context.getString(resources.getIdentifier(
-                APPLICATION_ID, STRING, packageName));
-        Crittercism.initialize(context, crAppID);
+        int identifier = resources.getIdentifier(APPLICATION_ID, STRING, packageName);
```

Comment:
More than double-initialization, I wonder if this "old initialization" runs before the customer calls the new Crittercism.init(). So if they already have some app ID in their resources, will they be able to change it with "new initialization"?